pkgs/by-name/mu/musl/package.nix

```diff
@@ -3,6 +3,7 @@
   stdenvNoLibc,
   lib,
   fetchurl,
+  fetchpatch,
```

Comment:
```suggestion
```

Reply:
oops, that slipped through from experiments, fixed